components/private/ThumbTrack/src/MDCThumbTrack.h

```diff
@@ -26,21 +26,46 @@
 /** The delegate for the thumb track. */
 @property(nullable, nonatomic, weak) id<MDCThumbTrackDelegate> delegate;
 
-/** The color of the thumb and left track. */
-@property(nullable, nonatomic, strong) UIColor *primaryColor;
-
-/** The color of the thumb off color. */
-@property(nullable, nonatomic, strong) UIColor *thumbOffColor;
+/**
+ The color of the thumb when enabled.
 
-/** The color of the track off color. */
-@property(nullable, nonatomic, strong) UIColor *trackOffColor;
+ Defaults and resets to blue.
+ */
+@property(null_resettable, nonatomic, strong) UIColor *thumbEnabledColor;
```

Comment:
Please add back `thumbOffColor` so that we don't introduce a breaking change in the next release.

Reply:
It's a property that does nothing and isn't used internally. Are you sure you want to keep it?

Reply:
Ah if nobody is using it then nevermind.